README.rst

```diff
@@ -29,9 +29,9 @@ What?
 Why?
 ====
 
-Working with large remote files, for example using Amazon's `boto <http://docs.pythonboto.org/en/latest/>`_ and `boto3 <https://boto3.readthedocs.io/en/latest/>`_ Python library, is a pain.
-``boto``'s ``key.set_contents_from_string()`` and ``key.get_contents_as_string()`` methods only work for small files, because they're loaded fully into RAM, no streaming.
-There are nasty hidden gotchas when using ``boto``'s multipart upload functionality that is needed for large files, and a lot of boilerplate.
+Working with large remote files, for example using Amazon's `boto3 <https://boto3.readthedocs.io/en/latest/>`_ Python library, is a pain.
+``boto3``'s ``key.set_contents_from_string()`` and ``key.get_contents_as_string()`` methods only work for small files, because they're loaded fully into RAM, no streaming.
```

Comment:
Does boto3 even have `set_contents_from_string` / `get_contents_as_string `?

Reply:
Ah good point, I don't think it does. `boto3` has `upload_fileobj()` and `download_fileobj()` which do handle multipart streaming. The point about having a unified pythonic API still stands though obviously.